js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadHooksContext.tsx

```diff
@@ -12,6 +12,17 @@ type LaunchpadHooksContextValue = {
   LaunchRootExecutionButton?: typeof LaunchRootExecutionButton;
   useLaunchWithTelemetry?: typeof useLaunchWithTelemetry;
   UserDisplay?: typeof UserDisplay;
+  useAllUserDetails?: () => {
```

Comment:
We have so many contexts that do this kind of injection 😅 . I would recommend `CloudOSSContext` but uhh it really doesnt matter (Right now). In the future it would matter for page split bundling but there would be other things to figure out then anyways

Reply:
Oh good call, I was blindly following `UserDisplay` 😅